rwslib/builders/clinicaldata.py

```diff
@@ -399,7 +399,7 @@ def __init__(
         :param str itemoid: OID for the matching :class:`ItemDef`
         :param str value: Value for the the ItemData
         :param str specify_value: 'If other, specify' value - *Rave specific attribute*
-        :param str transaction_type: Transaction type for the data
+        :param str transaction_type: Transaction Type for Data (one of **Insert**, **Update**, **Upsert**, **, Context**, **Remove**)
```

Comment:
I think the `**` are in the wrong place here

Reply:
gah, thanks.  New push to address.